projects/vdk-plugins/vdk-jobs-troubleshooting/README.md

```diff
@@ -1 +1,245 @@
-### VDK-JOBS-TROUBLESHOOTING Plugin
+## VDK-JOBS-TROUBLESHOOTING Plugin
+
+The VDK JOB Troubleshooting plugin provides the ability to add various troubleshooting utilities which can be accessed
+during the data job runtime.
+
+Generally it's quite hard to produce a thread dump of a python process running inside a kubernetes pod. So this plugin
+provides a thread-dump utility
+
+### Adding the plugin to your data job
+
+Before you can use the plugin you should add it to your data job or custom-sdk, by adding the following line to your
+requirements.txt file:
+
+```commandline
+install_requires = vdk-jobs-troubleshooting
```

Comment:
This is not in the requirements.txt file. `install_requires` is part of the setup.py/setup.cfg files. For the requirements.txt, specifying only the name of the plugin is enough.

Reply:
fixed.